core/sr-version/src/lib.rs

```diff
@@ -228,9 +228,8 @@ mod apis_serialize {
 	pub fn deserialize_bytes<'de, D>(d: D) -> Result<super::ApiId, D::Error> where
 		D: de::Deserializer<'de>
 	{
-		let bytes = bytes::deserialize_check_len(d, bytes::ExpectedLen::Exact(8))?;
 		let mut arr = [0; 8];
-		arr.copy_from_slice(&bytes);
+		let _ = bytes::deserialize_check_len(d, bytes::ExpectedLen::Exact(&mut arr[..]))?;
```

Comment:
I am not familiar with this code, why we do this change? Does it have anything to do with new dependencies?

Reply:
It is due to an upstream API change in `impl-serde`, if I recall correctly.